src/iuse.cpp

```diff
@@ -462,9 +462,9 @@ static hp_part body_window(player *p, item *, std::string item_name,
 }
 
 // returns true if we want to use the special action
-static hp_part use_healing_item(player *p, item *it, int normal_power, int head_power,
-                                int torso_power, int bleed,
-                                int bite, int infect, bool force)
+hp_part use_healing_item(player *p, item *it, int normal_power, int head_power,
```

Comment:
Yep - I use it to cauterize from `iuse_actor.cpp`. Currently as `extern`, but it would probably work better if all the invocations were actually `iuse_actor` type - medical item functions are simply checking for under water and then invoking `use_healing_item`. Except for first aid kit, which also assigns an activity.